src/stats.rs

```diff
@@ -0,0 +1,35 @@
+use std::borrow::Borrow;
+use api::{Id, Collection, Bool, OwnerId};
+use serde::de;
+use std::fmt::Debug;
+use std::str::FromStr;
+use chrono::naive::date::NaiveDate;
```

Comment:
Consider making this import `pub use`, so our users won't need to depend on chrono directly.